docs/src/functions_list.md

```diff
@@ -59,4 +59,10 @@ SpecialFunctions.beta
 SpecialFunctions.logbeta
 SpecialFunctions.logabsbeta
 SpecialFunctions.logabsbinomial
+SpecialFunctions.chebyshevT
```

Comment:
All function names should be lower case (`chebyshevt`, `chebyshevu`, `hermiteh`, etc.) for consistency with Julia conventions.

Reply:
done. see commit [0fbd56a](https://github.com/JuliaMath/SpecialFunctions.jl/pull/175/commits/0fbd56a6daab5b68952e63c645f5450ac62e8f1e)